src/haskell/install.sh

```diff
@@ -1,23 +1,29 @@
 #!/usr/bin/env bash
 set -e
 
+GHCUP_VERSION="0.1.19.2"
```

Comment:
Wondering, is there a way to automatically find the latest version for `GHCUP` instead of pinning this? We would like to remove such dependencies that needs manual updates (if possible) 

eg. For git releases we have helper functions which looks at git tags and finds the latest version, hence we are always up-to-date without modifying scripts.

Doesn't look like https://github.com/haskell/ghcup/tags has tags updated. 😕

Reply:
We have to pin on a version if we want to do SHA256SUM validation. Otherwise the attacker can just change both the download and the SHA256SUM.

Reply:
Makes sense 👍

Reply:
> Doesn't look like [haskell/ghcup/tags](https://github.com/haskell/ghcup/tags) has tags updated. 😕

It seems to refer to an old repository. The new repository have tags.
https://github.com/haskell/ghcup-hs